common/versioning/version_tasks.gradle

```diff
@@ -34,6 +45,18 @@ private void saveChanges(String versionName) {
     versionProps.store(getVersionFile().newWriter(), null)
 }
 
+private void incrementLatestVersion(){
+    def versionProps = getVersionProps()
+    versionProps["latestPatchVersion"] = (getLatestPatchVersion() + 1).toString()
```

Comment:
Does groovy coerce this to 0?

Reply:
I assume you're referring to type coercion, but not sure i'm following in this case.  getLatestPatchVersion returns an integer or it throws an exception and would fail the script.

Reply:
> I assume you're referring to type coercion, but not sure i'm following in this case. getLatestPatchVersion returns an integer or it throws an exception and would fail the script.

I'm just wondering about the case where that file is empty or doesn't exist.  We implicitly accept that earlier.  I'd rather fail there.

Reply:
I didn't write that code, and i'd like to avoid refactoring something that doesn't have any record of being broken.